pkg/drivers/qemu/qemu.go

```diff
@@ -445,7 +445,9 @@ func (d *Driver) Start() error {
 			d.diskPath())
 	}
 
-	if stdout, stderr, err := cmdOutErr(d.Program, startCmd...); err != nil {
+	socketCmd := append([]string{"/var/run/socket_vmnet", d.Program}, startCmd...)
```

Comment:
You have to provide the socket path
https://github.com/lima-vm/socket_vmnet#usage

Reply:
I think we are providing the path?

Client: `"/opt/socket_vmnet/bin/socket_vmnet_client"`
Path: `/var/run/socket_vmnet`

On line 450, we take the client and append the socket path along with other arguments.

Reply:
Ah, yes, overlooked “/var/run” sorry

Reply:
Sure, I appreciate your review.